nautobot/ipam/tables.py

```diff
@@ -771,7 +771,7 @@ class InterfaceVLANTable(StatusTableMixin, BaseTable):
     tenant = TenantColumn()
     role = tables.TemplateColumn(template_code=VLAN_ROLE_LINK)
     location_count = LinkedCountColumn(
-        viewname="dcim:location",
+        viewname="dcim:location_list",
```

Comment:
Could we add an assertion in `LinkedCountColumn.__init__()` that would have caught this error? Or would that be too prescriptive for app usage?

Reply:
It would probably be nice to catch this during development instead of after release. If an app needs to overload it they can?

Reply:
Looks like we can't do this on init because Django hasn't resolved all URLs yet